### ISSUE TITLE
Upgrading Evo does not retain updated plugin enabled/disabled status

### ISSUE THREAD

**Issue by bossloper** [evolution-cms/evolution#922]
When upgrading if a plugin has a new version then it enables it instead of retaining the enabled/disabled status of the previous version.

e.g. I have a 1.4.6 evo install with both: 
userHelp (1.8.0)
ElementsInTree (1.5.9)
set to disabled.

After upgrading to 1.4.7 both:
UserHelp (1.8.1)
ElementsInTree (1.5.10)
are now enabled.

**Comment by bossloper** [comment]
just updated 2 more from 1.4.6 to 1.4.7 and both enabled EIT. Are you sure your EIT was updated and was not already at version 1.5.10?
![untitled](https://user-images.githubusercontent.com/1744687/49742654-dee77100-fc90-11e8-927f-f9e03309f3f7.png)

**Comment by dmi3yy** [comment]
By default EIT is OFF 
https://github.com/evolution-cms/evolution/blob/1.4.x/install/assets/plugins/ElementsInTree.tpl#L15

**Comment by dmi3yy** [comment]
Yes, when updating a version, a new plug-in is specially created in order that nothing is lost and there is also a button to delete old plug-ins. Ie current behavior has not changed

**Comment by bossloper** [comment]
If I delete a disabled EIT plugin before upgrading then yes the new EIT plugin is disabled.
But if the plugin exists in the install (disabled) then the system upgrade makes the new version of EIT enabled.
